specification/sql/resource-manager/Microsoft.Sql/preview/2023-05-01-preview/JobAgents.json

```diff
@@ -290,13 +302,52 @@
           "$ref": "../../../common/v1/types.json#/definitions/Sku",
           "description": "The name and tier of the SKU."
         },
+        "identity": {
+          "$ref": "#/definitions/JobAgentIdentity",
+          "description": "The identity of the job agent."
+        },
         "properties": {
           "$ref": "#/definitions/JobAgentProperties",
           "description": "Resource properties.",
           "x-ms-client-flatten": true
         }
       }
     },
+    "JobAgentIdentity": {
```

Comment:
>JobAgentIdentity [](http://example.com/codeflow?start=5&length=16)

It's best if we can reuse the common-types definitions for managed identity....

Reply:
Please replace with references to appropriate definitions from `[https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/managedidentity.json`](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/managedidentity.json`) or reply to explain any problems with that.

Reply:
Hi Tim, this was added by merging with new AAD changes, @rayboyd30 did managed identity presented any problems?

Reply:
this is the other PR changes
https://github.com/Azure/azure-rest-api-specs/pull/25553/files#diff-269ee4de58bde0815574778fe9ccc75fdb419beedef3e04f7f5d791f15fe4b2b

Reply:
Looks like that PR hasn't been reviewed yet? Are you trying to club two reviews together here? It might in fact be simpler to get it finished that way...

Reply:
Seems like this would require extra effort to define common type for SQL. From other conversation:

[rayboyd30](https://github.com/rayboyd30) [5 days ago](https://github.com/Azure/azure-rest-api-specs/pull/25553/files#r1331943597)
aligning with SQL common type def needs extra efforts from SQL security team. We have to ask for an exception in current version. we will coordinate with SQL security team to fix it in next version. But for this version, please unblock.
Here is the work item to align common types: https://msdata.visualstudio.com/Database%20Systems/_workitems/edit/2697929

Reply:
please wirk with your team to move the identities schema into the SQL common types. It is not efficient to keep redifining it everywhere. Signing off on this for now.